### DISCUSSION TITLE
data is missing in self hosted nominatim

### DISCUSSION THREAD

**Discussion by MALKGITHUBER** [osm-search/Nominatim#3439] [Q&A]
Hi Team,
how to index the data if some data went missing or not indexed 

for example 09 Triệu Quốc Đạt,Điện Biên,Thanh Hóa,Thanh Hóa'

this is the data which is not present in my own nominatim instance but it is present in the [nominatim.openstreetmap.org](https://nominatim.openstreetmap.org/)

Kindly let me know is there any solution for the data missing issue

**Comment by mtmail** [comment]
You can use `nominatim add-data` for individual nodes, ways, relations or even whole files. Then run `nominatim index`.

If the data is already in Nominatim (for example use `nominatim details` to check for a node, way, relation), then you might also need to run `nominatim refresh --data-object` or `nominatim refresh --data-area` (and then `nominatim index`).

All commands have a `--help` parameter.
